common/src/main/java/org/opensearch/ml/common/transport/agent/MLAgentUpdateInput.java

```diff
@@ -105,6 +110,7 @@ public MLAgentUpdateInput(StreamInput in) throws IOException {
         Version streamInputVersion = in.getVersion();
         agentId = in.readString();
         name = in.readOptionalString();
+        type = streamInputVersion.onOrAfter(VERSION_3_4_0) ? in.readOptionalString() : null;
```

Comment:
Need to change this to
```suggestion
        type = streamInputVersion.onOrAfter(VERSION_3_3_0) ? in.readOptionalString() : null;
```